Anchor.toml

```diff
@@ -27,4 +27,4 @@ cluster = "localnet"
 wallet = "~/.config/solana/id.json"
 
 [scripts]
-test = "yarn run ts-mocha -t 1000000 tests/gummyroll.ts"
```

Comment:
I don't know if this change was on purpose or if this was accidentally committed when you were testing gummyroll alone.